src/SplitIO/Component/Log/Logger.php

```diff
@@ -57,7 +57,7 @@ public function __construct(LogHandlerInterface $handler = null, $level = LogLev
      * @param array $context
      * @return null
      */
-    public function log($level, $message, array $context = array())
+    public function log($level, $message, array $context = array()): void
```

Comment:
This should not work for psr/log in version 3.

v3 - https://github.com/php-fig/log/blob/3.0.0/src/LoggerInterface.php#L124
v1 - https://github.com/php-fig/log/blob/1.1.4/Psr/Log/LoggerInterface.php#L124

Interface declaration are different.